frontend/src/modules/contributor/components/edit/work-history/contributor-work-history-edit.vue

```diff
@@ -242,6 +254,27 @@ const isModalOpen = computed<boolean>({
   },
 });
 
+const hasSameOrganization = computed(() => props.contributor.organizations.some((o: Organization) => o.id === form.organization?.id)
+    && (!isEdit.value || form.organization?.id !== props.organization?.id));
```

Comment:
Should this "isEdit" also be removed? Since validation should be on both edit + add

Reply:
no here the isEdit is okey since that check can then cause that it checks also itself for the organization. That additional check is preventing that on edit the message would be always shown since it would check itself aswell.